### DISCUSSION TITLE
idea for advanced users

### DISCUSSION THREAD

**Discussion by Natejoestev** [ScratchAddons/ScratchAddons#3301] [Ideas]
there should be an option for `Ctrl + Space` that when you type something not found there is an option for
`create function "NAME"` like in Unity when you add a component

example:
press `Ctrl + Space`,
type `hello, world`,
select option `create function "hello, world"`
shows `make a block` window with the name equal to `"hello, world"` instead of default `"block name"`

good idea? it would be nice

also `Alt + left click` works

**Comment by apple502j** [comment]
Not sure if that's more useful than shortcuts for opening modals or creating blocks straight from keyboard.
